src/Lift.ts

```diff
@@ -216,55 +199,12 @@ export class Lift {
   public async recreateStudioServer(datamodel: string, providerAliases: { [key: string]: string }) {
     try {
       if (this.studioServer) {
-        this.studioServer.restart({ datamodel })
+        this.studioServer.restart(datamodel, providerAliases)
         return
       }
 
-      const platform = await getPlatform()
```

Comment:
All of this moved to: https://github.com/prisma/lift/pull/179/files#diff-7b9a013ccbe71a24d9d1034b8f94efa3